covid_moonshot_ml/datasets/utils.py

```diff
@@ -130,13 +131,32 @@ def cdd_to_schema(cdd_csv, out_json, achiral=False):
     df['pIC50'] = pic50_vals
     df['pIC50_range'] = pic50_range
 
+    ci_lower_key = ('ProteaseAssay_Fluorescence_Dose-Response_Weizmann: IC50 '
+        'CI (Lower) (µM)')
+    ci_upper_key = ('ProteaseAssay_Fluorescence_Dose-Response_Weizmann: IC50 '
+        'CI (Upper) (µM)')
+    ## Calculate 95% CI in pIC50 units based on IC50 vals (not sure if the
+    ##  difference should be taken before or after taking the -log10)
+    pic50_var = []
+    for _, (ci_lower, ci_upper) in df[[ci_lower_key, ci_upper_key]].iterrows():
+        if pandas.isna(ci_lower) or pandas.isna(ci_upper):
+            pic50_var.append(np.nan)
+        else:
+            ## First convert bounds from IC50 (uM) to pIC50
+            pic50_ci_upper = -np.log10(ci_upper*10e-6)
+            pic50_ci_lower = -np.log10(ci_lower*10e-6)
+            ## Assume size of 95% CI == 4*sigma => calculate variance from stdev
+            pic50_var.append(((pic50_ci_lower - pic50_ci_upper) / 4) ** 2)
```

Comment:
Suggest storing the `stderr` (standard error) is better than the `var` (variance).